testsuite/features/step_definitions/common_steps.rb

```diff
@@ -1372,10 +1372,75 @@ def server_secret
 And(/^I check the Cobbler parameter "([^"]*)" with value "([^"]*)" in the isolinux.cfg$/) do |param, value|
   tmp_dir = "/var/cache/cobbler/buildiso"
   result, code = $server.run("cat #{tmp_dir}/isolinux/isolinux.cfg | grep -o #{param}=#{value}")
-  raise "error during veryfying isolinux.cfg parameter for Cobbler buildiso.\nLogs:\n#{result}" if code.nonzero?
+  raise "error while verifying isolinux.cfg parameter for Cobbler buildiso.\nLogs:\n#{result}" if code.nonzero?
 end
 
 When(/^I cleanup after Cobbler buildiso$/) do
   result, code = $server.run("rm -Rf /var/cache/cobbler")
   raise "error during Cobbler buildiso cleanup.\nLogs:\n#{result}" if code.nonzero?
 end
+
+When(/^I reboot the server through SSH$/) do
+  init_string = "ssh:#{$server.public_ip}"
+  temp_server = twopence_init(init_string)
+  temp_server.extend(LavandaBasic)
+  temp_server.run('reboot > /dev/null 2> /dev/null &')
+  default_timeout = 300
+
+  check_shutdown($server.public_ip, default_timeout)
+  check_restart($server.public_ip, temp_server, default_timeout)
+
+  repeat_until_timeout(timeout: default_timeout, message: "Spacewalk didn't come up") do
```

Comment:
This message will be coming up all the time until the timeout, right? Then please change it to "Spacewalk hasn't come up yet" to make it clear it could come up later. I'm actually not sure of the definition of this function, if the message appears only once at the end of the failure to come up, then it should be ok as it is.

Reply:
It appears only once if it times out if I am not mistaken (for instance like in https://github.com/uyuni-project/uyuni/blob/master/testsuite/features/step_definitions/common_steps.rb#L151)